catalog/spatial/wfs/spatial-wfs-featuretransformer-handlebars/src/main/java/org/codice/ddf/spatial/ogc/wfs/transformer/handlebars/HandlebarsWfsFeatureTransformer.java

```diff
@@ -249,6 +250,18 @@ private String getIdAttributeValue(
     return id;
   }
 
+  private void addXmlAttributesToContextMap(
+      final StartElement startElement, final Map<String, String> contextMap) {
+    for (final Iterator attributeIterator = startElement.getAttributes();
```

Comment:
❓ Couldn't this be written as a stream iterator? Something like:
`
Iterable<Attribute> iterable = () -> startElement.getAttributes();
StreamUtils.stream(iterable.spliterator(), false)
         .map(attribute -> ...do map stuff...)
         .forEach(mapEntry -> contextMap.put(mapEntry.getKey(), mapEntry.getValue());
`
But even if not, the compound statement in the clause is a tad odd.

Reply:
It could, but it ends up looking like this:
```java
StreamSupport.stream(
        Spliterators.spliteratorUnknownSize(
            (Iterator<javax.xml.stream.events.Attribute>) startElement.getAttributes(),
            Spliterator.IMMUTABLE),
        false)
    .forEach(
        attribute -> {
          final String attributeKey =
              startElement.getName().getLocalPart() + "@" + attribute.getName().getLocalPart();
          contextMap.put(attributeKey, attribute.getValue());
        });
```

Reply:
I've slightly improved it (I think) by switching to a `while` loop instead of a `for`.